src/shared/services/unitsService.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+import Unit from '../models/unit';
+
+export const createUnit = (unit: Unit) => new Promise((resolve, reject) => axios
```

Comment:
Missing unit tests for the unitService :)

Reply:
good spot :) I've added them